src/components/vr_cooperation/include/vr_cooperation/vr_module.h

```diff
@@ -112,16 +117,41 @@ class VRModule : public functional_modules::GenericModule,
   void SendMessageToMobile(application_manager::MessagePtr msg);
 
   /**
-   * @brief receive message from mobile
-   */
-  void ReceiveMessageFromMobile();
-
-  /**
    * Handles received message from HMI (Applink)
    * @param message is GPB message according with protocol
    */
   virtual void OnReceived(const vr_hmi_api::ServiceMessage& message);
 
+  /**
+   * @brief Returns key of application that process VR requests.
+   * @return mobile application connection key
+   */
+  uint32_t activated_connection_key() const {
```

Comment:
it's coding style from CM project (https://github.com/CustomSDL/connectivity_manager/pull/55)